modules/cli-options/src/main/scala/scala/cli/commands/PackageOptions.scala

```diff
@@ -39,6 +39,12 @@ final case class PackageOptions(
   @HelpMessage("Generate an assembly JAR")
     assembly: Boolean = false,
   @Group("Package")
+  @HelpMessage("For assembly JAR, whether to add a bash / bat preamble")
+    preamble: Boolean = true,
+  @Group("Package")
+  @HelpMessage("Generate an assembly JAR for Spark")
```

Comment:
I think the doc should mention that it will exclude spark modules

Reply:
Should be fixed.